externals/simplecpp/simplecpp.cpp

```diff
@@ -20,13 +20,13 @@
 #include <cstring>
 #include <ctime>
 #include <exception>
-#include <fstream>
+#include <fstream> // IWYU pragma: keep
```

Comment:
These changes were missing upstream because I accidentally did them here in #5673. Addressed via https://github.com/danmar/simplecpp/pull/330.